libs/flowbite-angular/src/lib/components/navbar/navbar-icon-button.component.ts

```diff
@@ -0,0 +1,32 @@
+import type { DeepPartial } from '../../common';
+import { createClass } from '../../utils';
+import { BaseComponent } from '../base-component.directive';
+import type { NavbarIconButtonClass, NavbarIconButtonTheme } from './navbar-icon-button.theme';
+import { NavbarIconButtonThemeService } from './navbar-icon-button.theme.service';
+
+import { Component, inject, input, signal } from '@angular/core';
+
+@Component({
+  selector: 'flowbite-navbar-icon-button',
+  standalone: true,
+  imports: [],
+  template: `<ng-content />`,
+})
+export class NavbarIconButtonComponent extends BaseComponent {
+  public readonly themeService = inject(NavbarIconButtonThemeService);
+
+  public override contentClasses = signal<NavbarIconButtonClass>(createClass({ rootClass: '' }));
+
+  //#region properties
```

Comment:
Make input color on navbar colors, and its default value is either navbarcontent color or navbar color